bin/create-node-module-types.sh

```diff
@@ -0,0 +1,5 @@
+cat >cucumber-tsflow/dist/esm/package.json <<!EOF
+{
+    "type": "module"
+}
+!EOF
```

Comment:
Copied from https://github.com/jwalton/solox/blob/4d86f8779329ffc47552a16d2bab60544ee78c65/bin/build-types.sh. This creates a `package.json` file with `"type": "module"` in the esm directory that gives a hint to nodejs that it doesn't need to use it commonjs loader to translate these files into esm.